identify/identify.py

```diff
@@ -85,7 +86,7 @@ def tags_from_path(path: str) -> set[str]:
 
 def tags_from_filename(path: str) -> set[str]:
     _, filename = os.path.split(path)
-    _, ext = os.path.splitext(filename)
+    ext = ''.join(Path(filename).suffixes)
```

Comment:
this is not acceptable

Reply:
Can you please say why? Because it's using `pathlib`?

Reply:
because it's a breaking change and in general incorrect